recipes/clumper/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "clumper" %}
+{% set version = "0.2.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/clumper-{{ version }}.tar.gz
+  sha256: fd24ad1d3dbeb7e3f9a05fad2be9ab81408cf39b3bc9549b793b7f23bde9878b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - clumper
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/clumper/
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add comment:

```yaml
# License file manually packaged. See https://github.com/koaning/clumper/pull/59
license_file: LICENSE
...
```

I've opened a PR to fix having to manually package the license file (https://github.com/koaning/clumper/pull/59). The comment is a visual reminder to check the status of whether the license file is in a future release at which point you can delete the comment and the manually packaged copy.